proxies/venue-graphql-proxy/README.md

```
@@ -55,39 +55,39 @@ GraphQL playground will run on http://localhost:4000/proxy/graphql
-### `yarn test`
+### `pnpm test`
 
 Launches the test runner
 
-### `yarn build`
+### `pnpm build`
 
 Builds the app for production to the `build` folder.
 
-### `yarn start`
+### `pnpm start`
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)